docsrc/source/user_guide/qronos.rst

```diff
@@ -0,0 +1,367 @@
+Post-Training Quantization with Qronos
+=================================================================
+
+Qronos is a new post-training quantization (PTQ) algorithm that sequentially rounds 
+and updates neural network weights. It explicitly corrects quantization errors in 
+both the weights and activations of previous layers while diffusing error into future 
```

Comment:
`It explicitly corrects quantization errors in both the weights and activations`

I understand the meaning but it could be misleading. I would say "It accounts for quantization errors introduced by both weights and activations" or something like that